controllers/openshift_controller.go

```diff
@@ -1667,10 +1667,6 @@ const (
 // will be returned.
 func (r *KataConfigOpenShiftReconciler) updateStatus() error {
 
-	if r.getInProgressConditionValue() != corev1.ConditionTrue {
-		return nil
-	}
-
```

Comment:
Regarding the commit message, why does just editing KataConfig externally trigger this problem?  I'd say predating PR #329 should be the only scenario.

Reply:
Is  un-rebuilding nodes a blocker?

Reply:
> Is un-rebuilding nodes a blocker?

Not really as it doesn't prevent the operator to be functional.

Reply:
Oh I suspect there would be quite a lot of things that user could do to sabotage the controller which wouldn't recover. ;-) But that's a fact indepedent of the idea of this PR - in fact, this has always been true and continues to be true even after this PR I believe.

My idea was not to mix independent facts in the message and not put them in the same context as if they were related since that could confuse a future reader. I'm not insisting though.

Reply:
Ah this is merged already... never mind.